src/Microsoft.Health.Fhir.Core/Configs/HostingBackgroundServiceQueueItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.Core.Features.Operations;
+
+namespace Microsoft.Health.Fhir.Core.Configs;
+
+public class HostingBackgroundServiceQueueItem
+{
+    public QueueType Queue { get; set; }
+
+    public bool EnableHeartbeats { get; set; }
```

Comment:
Nit: This setting enables heavy heartbeats, all jobs have regular heartbeats enabled.

Reply:
Is the suggestion to update the field or add a comment?